src/utils/markets.js

```diff
@@ -280,6 +280,28 @@ export function _useUnfilteredTrades(limit = 10000) {
   //   .map(market.parseFillEvent.bind(market));
 }
 
+export function useBonfidaTrades() {
+  const { baseCurrency, quoteCurrency } = useMarket();
+  async function getBonfidaTrades() {
+    if (!baseCurrency || !quoteCurrency) {
+      return null;
+    }
+    const response = await fetch(
+      `https://serum-api.bonfida.com/trades/${baseCurrency + quoteCurrency}`,
```

Comment:
Should pass in a market address; might need to chat with them about supporting this